dev-tools/docker/docs/Dockerfile

```diff
@@ -14,12 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ruby:2.3.1
+FROM ruby:2.5.3
```

Comment:
Just as an fyi, the latest (and last) release of 2.5.x is 2.5.9.  Of course this being ruby, I would not be shocked to find out exactly 2.5.3 is needed for this to all work

Reply:
Does Docker allow you to specify greater-than-or-equal (>=) with a version number? That would be ideal. If not, 2.5.3 is a good working minumum version with the current toolset.

Reply:
You can use `FROM ruby:2.5` to get the latest ruby 2.5 version.  I wouldn't block merging the PR on this.  Ruby 2.5 is end of life anyway the real answer is use some tools that work with later ruby versions

Reply:
I tried using `ruby:2.5` as base image. As it works fine, I have updated the Dockerfile. Anyway, as @moleske pointed out, Ruby 2.5 is dead. Is anyone working actively on making bookbinder to work on Ruby 2.7?